modules/wolfgame.py

```diff
@@ -787,6 +795,189 @@ def fpinger(cli, nick, chan, rest):
     var.LAST_PING = None
     pinger(cli, nick, chan, rest)
 
+@cmd("pingif", "pingme", "pingat", pm=True)
+def altpinger(cli, nick, chan, rest):
+    """Pings you when the number of players reaches your preference."""
+    if not rest or (rest.isdigit() and int(rest) == 0):
+        if is_user_altpinged(nick)[0]:
+            msg = "Your ping preferences have been removed (was {0}).".format(is_user_altpinged(nick)[1])
+            if chan == nick:
+                pm(cli, nick, msg)
+            else:
+                cli.notice(nick, msg)
+            toggle_altpinged_status(nick, 0, is_user_altpinged(nick)[1])
+        elif not rest:
+            if chan == nick:
+                pm(cli, nick, "You need to enter a number.")
+            else:
+                cli.notice(nick, "You need to enter a number.")
+        else:
+            if chan == nick:
+                pm(cli, nick, "You do not have any preferences set.")
+            else:
+                cli.notice(nick, "You do not have any preferences set.")
+    elif rest.isdigit():
+        rest = int(rest)
+        if rest >= var.MAX_PLAYERS:
```

Comment:
var.MAX_PLAYERS changes based on gamemode (or can change). If you absolutely MUST limit it, I'd rather go for some arbitrarily high number like 9999. There is basically no downside to picking a number higher than how many people would feasibly join as you'd just never get pinged.